artifacts.go

```diff
@@ -56,14 +59,42 @@ func artifactsHandler(packagesBasePaths []string, cacheTime time.Duration) func(
 		w.Header().Set("Content-Type", "application/gzip")
 		cacheHeaders(w, cacheTime)
 
-		err = archiver.ArchivePackage(w, archiver.PackageProperties{
+		properties := archiver.PackageProperties{
 			Name:    packageName,
 			Version: packageVersion,
 			Path:    packagePath,
-		})
-		if err != nil {
-			log.Printf("archiving package path '%s' failed: %v", packagePath, err)
+		}
+		// TODO: find proper directory, make it a config option
+		basePath := "/tmp/epr"
+		path := basePath + "/" + properties.Name + "-" + properties.Version + ".tar.gz"
+
+		// TODO: Add dev option to skipt caching part
```

Comment:
I'd like to see this condition implemented. You can use devmode package, there are some methods exposed.